src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -654,6 +655,23 @@ export class ResultsViewPageStore {
                             ret.push(profile);
                         }
                     }));
+                } else if (profile.molecularAlterationType === AlterationTypeConstants.GENESET_SCORE) {
+                    // geneset profile, we dont have the META projection for geneset data, so just add it
```

Comment:
kill comment?

Reply:
no I think leave because we'll want to change that if we ever add that projection